git-filter-repo

```diff
@@ -3542,7 +3542,7 @@ class RepoFilter(object):
       self._output = open(self._fe_filt, 'bw')
     else:
       self._output = self._fip.stdin
-    if self._args.debug:
+    if self._args.debug and self._fip:
```

Comment:
I would rather this check 'not self._args.dry_run' rather than 'self._fip'; while both may be valid, I think the former documents the intent a bit better to future readers.

Reply:
No problem. I see how that makes the intent a little easier to follow. I rebased my branch on the latest `master`, made that one small change, and verified that all of the tests still pass.